vulkan_helpers/vulkan_application.cpp

```diff
@@ -740,6 +740,10 @@ VulkanApplication::CreateAndBindHostBuffer(
 containers::unique_ptr<VulkanApplication::Buffer>
 VulkanApplication::CreateAndBindCoherentBuffer(
     const VkBufferCreateInfo* create_info, const uint32_t* device_indices) {
+    // If this goes off, you have to update the bits when we create the coherent heap.
+    //   Be careful not every implementation supports every usage. They all seem to
+    //   support this usage.
```

Comment:
nit: comment indentation style seems inconsistent with your other multiline comment (which I also commented on, but not for style).